lms/services/async_oauth_http.py

```diff
@@ -0,0 +1,60 @@
+import asyncio
+from typing import List
+
+import aiohttp
+
+from lms.services import ExternalRequestError
+
+
+class AsyncOAuthHTTPService:
```

Comment:
Just one service, in theory we could have both a `AsyncHTTPService` and build `AsyncOAuthHTTPService` on top. In practice we only have one usage for this so that might be too much for now.